src/app_state.rs

```diff
@@ -1,69 +1,28 @@
 use crate::errors::Err;
-use entropy_client::chain_api::{EntropyConfig, get_api, get_rpc};
+use entropy_client::forest::TreeState;
 use serde::Deserialize;
-use sp_core::{Pair, crypto::AccountId32, sr25519};
 use std::{
     collections::HashMap,
     sync::{Arc, RwLock},
 };
-use subxt::{
-    OnlineClient, backend::legacy::LegacyRpcMethods, utils::AccountId32 as SubxtAccountId32,
-};
-use x25519_dalek::StaticSecret;
 
 /// Application state struct which is cloned and made available to every axum HTTP route handler function
 #[derive(Clone)]
 pub struct AppState {
-    /// Keypair for box id account
-    pub pair: sr25519::Pair,
-    /// Secret encryption key
-    pub x25519_secret: StaticSecret,
-    /// Configuation containing the chain endpoint
-    pub configuration: Configuration,
+    /// Tree state from entropy-client
```

Comment:
This could be more descriptive. If it's important to indicate this is coming from the client crate we can also just use it directly, e.g `entropy_client::forest::TreeState`